app/javascript/controllers/tooltip_controller.js

```diff
@@ -0,0 +1,82 @@
+import { Controller } from '@hotwired/stimulus'
+import {
+  computePosition,
+  flip,
+  shift,
+  offset,
+  arrow
+} from '@floating-ui/dom';
+
+export default class extends Controller {
+  static targets = ["element", "tooltip"];
+  static values = {
+    placement: String,
+    offset: Number,
+    crossAxis: Number
+  };
+
+  initialize() {
+    this.placementOption = this.placementValue || "top"
+    this.offsetOption = this.offsetValue || 10
+    this.crossAxisOption = this.crossAxisValue || 0
+    this.alignmentAxisOption = this.alignmentValue || null
+    this.arrowElement = document.querySelector('#arrow');
+  }
+
+  connect() {
+    this.elementTarget.addEventListener("mouseenter", this.showTooltip);
+    this.elementTarget.addEventListener("mouseleave", this.hideTooltip);
+    this.elementTarget.addEventListener("focus", this.showTooltip);
+    this.elementTarget.addEventListener("blur", this.hideTooltip);
+  }
+
+  showTooltip = () => {
+    tooltip.style.display = 'block';
+    this.#update();
+  }
+
+  hideTooltip = () => {
+    tooltip.style.display = '';
```

Comment:
Here and in `showTooltip`, we're calling `tooltip` as a global variable.  This works because there is an element in the DOM with an id of `tooltip`, but creates a tight coupling between the view and this controller.  We'll eventually need to support multiple tooltips on the same page, so this will break when we do that.

I think we should use `this.tooltipTarget` here for stability moving forward

Reply:
[62fd745](https://github.com/maybe-finance/maybe/pull/1065/commits/62fd7453c6e876245bb873dcd49083949e9b33ae)